aspnetcore/migration/mvc.md

```diff
@@ -143,8 +143,6 @@ Upon saving the file, Bower will automatically download the dependencies to the
 
 See [Manage Client-Side Packages with Bower](../client-side/bower.md) for more information.
```

Comment:
[Bower](http://bower.io) is no longer used or recommended.

Reply:
So should bower be removed from all the samples and docs, or just remove this link?

Reply:
Yeah, we shouldn't be telling anyone to add Bower to a project.

Reply:
Not sure what the result should be. All of these samples have Bower in them. Since Bower is deprecated, removing all the bower content from the docs and the samples is ideal. The issue is all the templates have bower in them. On this doc, should we put a note about it. Just not sure what this change on this doc should be. Maybe update the doc to get bootstraps and jQuery from CDN?

Reply:
@scottaddie done